tests/installation/prepare_maintenance_env.pm

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Prepare test repositories for children jobs.
```

Comment:
other teams seems to have its own `patch_and_reboot.pm`, we don't need to invent new name and the reboot should be in the same file, because `tests/shutdown/reboot.pm` as I mentioned to you offline sounds very generic, each arch does things on its way and contributors will complain or would not know exact use case of this.
The trend now is to have each squad their own file, in the future there are some plan to organize the repo, so that is yet another reason.

Reply:
existing summary description was more clear:
```
# Summary: QA Automation: patch the system before running the test
#          This is to test Test Updates
```
Test files do not care about children jobs, that is an openQA setting/relationship.
Subsequent tests could be run right after the installation and not in child jobs for example.